### PR TITLE
Refactor the business logic out of the ASP.NET Core application

### PR DESCRIPTION
**Pull request by gldraphael**

### CODE REVIEW

**Comment by gldraphael** [comment]
We're not complete yet, but this is a good start. @losviko could you please review and merge this if it looks good to you?

**Comment by gldraphael** [comment]
Adding the WIP again. Need to fix migrations first.

File: src/EventManagement.Infrastructure/ApplicationDbContext.cs [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+            // Add your customizations after calling base.OnModelCreating(builder);
+
+        }
+
+        public DbSet<ApplicationUser> ApplicationUsers { get; set; }
+        public DbSet<EventInfo> EventInfos { get; set; }
+        public DbSet<Registration> Registrations { get; set; }
+        public DbSet<PaymentMethod> PaymentMethods { get; set; }
+        public DbSet<Product> Products { get; set; }
+        public DbSet<ProductVariant> ProductVariant { get; set; }
```
Comment (losviko) [inline src/EventManagement.Infrastructure/ApplicationDbContext.cs]: This is my mistake, but it should be ProductVariants. We´ll take that in the next iteration..

Comment (gldraphael) [reply src/EventManagement.Infrastructure/ApplicationDbContext.cs]: Yes, let's change it after this PR is merged.

**Pull request merged**